ansible/roles/passwords/defaults/main.yml

```diff
@@ -12,4 +12,7 @@ slurm_appliance_secrets:
 secrets_openhpc_mungekey_default:
   content: "{{ lookup('pipe', 'dd if=/dev/urandom bs=1 count=1024 2>/dev/null | base64') }}"
 
+k3s_secrets:
```

Comment:
I thought maybe it shouldn't be there since the ansible doesn't interact with it after this and it made the templating slightly nice but it works as a key too